doc/source/whatsnew/v0.22.0.txt

```diff
@@ -13,7 +13,7 @@ version.
 New features
 ~~~~~~~~~~~~
 
--
+- Simultaneous melting of independent groups of columns is now possible with ``melt``.
```

Comment:
you will need a full example, pls pointers to the docs. but let's settle on an API.

Reply:
Added Sentence to 'highlights' section and two examples to 'New Features' section.

API will be backwards compatiable. 
* `value_vars` can take a list of lists
* `value_name` and `var_name` can take a list of new column names

@TomAugspurger mentioned the addition of `keep_index` parameter. This can easily be added.

`wide_to_long` funcitonality can be exactly duplicated (and more) with a single additional parameter that takes a tuple of `suffix` and `sep` but this might be too much for one function. 

Regardless, `wide_to_long` can be refactored to be faster.

This new melt enhancement makes lreshape obsolete (other than it being faster)